xwiki-authenticator-trusted-api/src/main/java/org/xwiki/contrib/authentication/TrustedAuthenticationAdapter.java

```diff
@@ -65,6 +67,16 @@
      */
     boolean isUserInRole(String role);
 
+
+    /**
+     * @return the roles the user is part of.
+     * @since 1.5.2
+     */
+    default Collection<String> getUserRoles()
+    {
+        return null;
```

Comment:
Maybe add some javadoc regarding `null` since, afaics it has a special treatment